NitroxServer/Communication/Packets/Processors/DockingStateChangeProcessor.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+
+namespace NitroxServer.Communication.Packets.Processors
+{
+    class DockingStateChangeProcessor : AuthenticatedPacketProcessor<DockingStateChange>
+    {
+        private readonly PlayerManager playerManager;
+
+        public DockingStateChangeProcessor(PlayerManager playerManager)
+        {
+            this.playerManager = playerManager;
+        }
+
+        public override void Process(DockingStateChange packet, Player player)
+        {
+            playerManager.SendPacketToOtherPlayers(packet, player);
+        }
+    }
```

Comment:
Is this really needed? Isn't this the default behavior of the server  if no processor is there? There a few instances of this in your files.